types/iso-3166-1-alpha-2/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for iso-3166-1-alpha-2 1.0
+// Project: https://github.com/georgzoeller/iso-3166-1-alpha-2
+// Definitions by: Justus Dietrich <https://github.com/Justus-D>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function getCountry(code: string): string | undefined;
+export function getCode(country: string): string | null;
+export function getCountries(): string[];
+export function getCodes(): string[];
+export function getData(): { [code: string]: string | undefined };
```

Comment:
you can do this, as well, to export a symbol (as in original module), but that is optional:
```suggestion
declare class Iso31661a2 {
    getCountry(code: string): string | undefined;
    getCode(country: string): string | null;
    getCountries(): string[];
    getCodes(): string[];
    getData(): { [code: string]: string | undefined };
}

declare const singleton: Iso31661a2;
export = singleton;
```